### ISSUE TITLE
No simple way to take the mean of a Kayak object?

### ISSUE THREAD

**Issue by kswersky** [HIPS/Kayak#15]
Based on some advice from an avid Kayak user, I found that this was one way to do it:

invN = kayak.Constant(np.atleast_2d(1.0/y.shape[0]))
self.loss = kayak.MatMult(y,invN)

Something like this really should be a basic op.
